soccer/planning/InterpolatedPath.hpp

```diff
@@ -21,7 +21,7 @@ class InterpolatedPath : public Path {
     /// Each entry in InterpolatedPath is a MotionInstant and the time that the
     /// robot should be at that position and velocity.
     struct Entry {
-        Entry(MotionInstant inst, float t) : instant(inst), time(t) {}
+        explicit Entry(MotionInstant inst, float t) : instant(inst), time(t) {}
```

Comment:
This is fine, but I think that the `explicit` keyword is only relevant for single-parameter constructors.  I don't think it actually changes anything here.

http://en.cppreference.com/w/cpp/language/explicit